sdk/ml/azure-ai-ml/azure/ai/ml/operations/_datastore_operations.py

```diff
@@ -220,3 +227,84 @@ def create_or_update(self, datastore: Datastore) -> Datastore:
                 log_and_raise_error(ex)
             else:
                 raise ex
+
+    @monitor_with_activity(logger, "Datastore.Mount", ActivityType.PUBLICAPI)
+    @experimental
+    def mount(
+        self,
+        path: str,
+        mount_point: str = None,
+        mode: str = "ro_mount",
+        debug: bool = False,
+        persistent: bool = False,
+        **_kwargs,
+    ) -> None:
+        """Mount a datastore to a local path.
```

Comment:
Can you add an extra clause or two that explains why a mounted datastore path is useful? For non-CRUD operations, I think it's helpful to give context to the operations we surface.

Reply:
Added: Mount a datastore to a local path, _so that you can access data inside it
        under a local path with any tools of your choice._